org.lflang/src/org/lflang/generator/c/Actions.java

```diff
@@ -25,7 +25,7 @@
  * @author{Alexander Schulz-Rosengarten <als@informatik.uni-kiel.de>}
  * @author{Hou Seng Wong <housengw@berkeley.edu>}
  */
-public class CActionGenerator {
+public class Actions {
```

Comment:
**These renamings are totally a stylistic choice which I am happy to revert.**

Reasoning: Having all the files in `org.lflang.generator.c` start with "C" and end with "Generator" is a waste of characters that makes it unnecessarily difficult to find the desired file. It also doesn't really make sense since some of these files can only loosely be considered as "generators" -- a suffix that serves more to comply with Java's tyrannical nounification conventions than to convey useful information.

Reply:
I agree with you, but I think it might be best not to bundle such refactoring in this PR. One advantage of the current naming scheme is that it's clear from the filename or class name immediately which target it is for, so there is no need to disambiguate. That said, I think that the naming you suggest here is better. Perhaps @edwardalee has an opinion about this?

Reply:
How about `CGenerator` and `PythonGenerator` though? Should we drop the prefix on those as well? Or move them out of the `c` and `python` package, respectively?

Reply:
While I agree that this seems like a waste of characters, it is a common style in Java projects. Also I second @edwardalee's comment about distinguishing tabs/buffers and also I think it is easier to find the specific file you are looking for when using a fuzzy search. So I lean towards keeping things as they are, but I don't have a strong opinion here.

However, I don't think this PR is the place to make such a style change. If we want to change the style, we should do so consistently in the entire code base (including the C++ and  Rust generators for instance), and also record it in our style guide (which we don't have yet, but we should add it to the draft).

Reply:
It seems like this change is both controversial and unimportant, so I'll just revert it.